include/libtorrent/torrent_handle.hpp

```diff
@@ -276,6 +276,14 @@ namespace aux {
 		// guaranteed to finish in the same order as you initiated them.
 		void read_piece(piece_index_t piece) const;
 
+		// This function sets the first piece and the last piece of the range
+		// for the piece picker. It starts downloading from the specified piece
+		// in sequential download mode, which it enables. If the torrent
+		// metadata has not been downloaded yet, then the function does nothing.
+		void set_sequential_range(piece_index_t first_piece, piece_index_t last_piece) const;
+		// This function overloads the above function and implicitly sets the
+		// last piece to the end of the file the first piece is set to.
```

Comment:
the way the documentation generation tool works is that it will create two separate entries for these function now. If you put the comments above both functions (and no comment in between), both functions will be part of the same entry, which I think would make more sense. Since they do the same thing.

Reply:
Ok I will do that